platform/ios/src/MGLMapView.h

```diff
@@ -1136,6 +1136,25 @@ IB_DESIGNABLE
 /**
  Returns an array of rendered map features that intersect with a given point,
  restricted to the given style layers.
+ 
+ This method may return all features from the specified layers. To filter
+ the returned features, use the
+ `-visibleFeaturesAtPoint:inStyleLayersWithIdentifiers:predicate` method. For more
+ information about searching for map features, see that method’s documentation.
+ 
+ @param point A point expressed in the map view’s coordinate system.
+ @param styleLayerIdentifiers A set of strings that correspond to the names of
+ layers defined in the current style. Only the features contained in these
+ layers are included in the returned array.
```

Comment:
Tip: in Xcode, ⇧⌥⌘V pastes preserving indentation. (The same shortcut preserves formatting in any native macOS application.)